root/static/scripts/release-editor/fields.js

```diff
@@ -759,10 +759,11 @@ class Barcode {
             return false;
         }
 
-        for (var i = 0, calc = 0; i < 12; i++) {
+        let calc = 0;
+        for (let i = 0; i < 12; i++) {
             calc += parseInt(barcode[i]) * this.weights[i];
         }
-
```

Comment:
I think you accidentally added an indentation here

Reply:
Hi @joker314 , thanks for pointing that out i'll fix it right away :DD